v3/tutorials/08-ink-workshop/a-getting-started/index.mdx

```diff
@@ -130,10 +130,10 @@ The `cargo-contract` package provides a command-line interface for working with
 
 1. Open a terminal shell on your computer.
 
-1. Install `dylint-link`, required to lint ink! contracts, warning you about things like using API's in a way that could lead to security issues.
+1. Install `cargo-dylint` and `dylint-link`, required to lint ink! contracts, warning you about things like using API's in a way that could lead to security issues.
```

Comment:
This was not explicitly needed for me, it was for you?

Reply:
Just ran all the installation steps in a Docker container with a fresh `ubuntu:18.04` image and got this:
```
root@gontijo:/opt/flipper# cargo +nightly contract build
 [1/5] Checking ink! linting rules
ERROR: cargo-dylint was not found!
Make sure it is installed and the binary is in your PATH environment.

You can install it by executing `cargo install cargo-dylint`.root@gontijo:/opt/flipper# cargo +nightly contract build
 [1/5] Checking ink! linting rules
ERROR: cargo-dylint was not found!
Make sure it is installed and the binary is in your PATH environment.

You can install it by executing `cargo install cargo-dylint`.
```

I also see that dep in [cargo-contract#installation](https://github.com/paritytech/cargo-contract#installation).

Reply:
yes I know it's specified upstream, but I don't tihnk it's explicitly needed cc @cmichi can you chime in here? 
Related to #1044